rMATS_P/prepare_stat_inputs.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import print_function
+
+import argparse
+import os
+import os.path
+import shutil
+import sys
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Modify the files from an rMATS output directory to prepare'
+        ' a new --od directory for running --task stat')
+    parser.add_argument(
+        '--new-output-dir',
+        required=True,
+        help='Path to the directory to be used for running --task stat')
+    parser.add_argument('--old-output-dir',
+                        required=True,
+                        help='Path to the directory of existing rMATS output')
+    parser.add_argument(
+        '--sample-1-indices',
+        required=True,
+        help='Comma separated list of replicates to be used as sample 1 in the'
+        ' new statistical comparison. Each replicate is identified by its'
+        ' 0-based index in the concatenation of replicates from samples 1 and 2'
+        ' in the --old-output-dir. Example: If --old-output-dir had 2 reps in'
+        ' sample 1 and 3 in sample 2 then --sample-1-indices 0,1,4 would select'
+        ' both reps from the old sample 1 and the last rep from the old sample'
+        ' 2 to be the new sample 1')
```

Comment:
Here I think **group** 1 and **group** 2 would be preciser than **sample** 1 and **sample** 2. Maybe replace the flag '--sample-1-indices' with '--group-1-indices' and adjust the description accordingly.